libs/api/tsconfig.spec.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "strict": false,
```

Comment:
Generally, yes, this would be dangerous. We had to add it though, to keep the current behaviour. We did the following:

- Because the root tsconfig.base.json did not include the "strict" setting (which defaults to `false`), Prisma generated wrong type definitions for `findUnique`. FindUnique returns the record or null, but Prisma generated a signature that always returned the record.
- We added `"strict": true` to tsconfig.base.json: https://github.com/wepublish/wepublish/commit/bc0b48d712efba82054e4fca166391e4e4db582e#diff-0b280a445be167f54cd916c0718c952b8e0d4ca9621903d05eec25efd4c6ed6d
- We had to explicitely set `strict` to false in the `tsconfig.spec.json` (note: only for the specs), because the specs contained a huge amount of type errors.

So to summarize, the above change keeps the existing config for the `specs`, but allows us to enable strict checks for the rest of the code base.